### PR TITLE
feat: Fix wasm

### PR DESCRIPTION
**Pull request by markphelps**
Fixes: #94

Finally get flipt-client-browser SDK to work for both commonJS and ESM using https://rollupjs.org/

This also moves the WASM code back to `flipt-engine-wasm` as I think we can also leverage this to redo the `flipt-client-node` SDK to remove FFI and swap to wasm

See: https://github.com/flipt-io/flipt/pull/2952

## TODO

- [ ] Still need to get the browser tests working, having trouble with Jest. But will do in another PR

### CODE REVIEW

**Comment by markphelps** [comment]
Cc @erka

File: flipt-client-browser/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import init, { Engine } from '../pkg';
+import init, { Engine } from '../dist/flipt_engine_wasm.js';
```
Comment (GeorgeMac) [inline flipt-client-browser/src/index.ts:1]: Was this the pesky fix?

**Review by GeorgeMac — APPROVED** [review]
Nice one guys!

File: flipt-client-browser/src/index.ts (hunk 1 above)
Comment (markphelps) [reply flipt-client-browser/src/index.ts:1]: Yeah pretty much

**Pull request merged**